main.go

```diff
@@ -91,12 +92,23 @@ func main() {
 		tableFlag       = flag.StringP("table", "t", "TABLE", "table name of the sql format")
 		templateFlag    = flag.StringP("template", "T", "", "Use template as input")
 	)
+	flag.StringVarP(&completionFlag, "completion", "C", "", "print bash/zsh completion function, pass shell as argument (\"bash\" or \"zsh\")")
+
```

Comment:
The problem was / is that my function takes a `string` and `StringP` returns a type `*String`. I couldn't work around this yet because `switch` does not work with pointers.